### DISCUSSION TITLE
How to point github repositories to new self-hosted renovate

### DISCUSSION THREAD

**Discussion by chandapukiran** [renovatebot/renovate#25903] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, select which platform you are using.

GitHub Enterprise Server

### What is your question?

Hi, 

We have two self-hosted renovate running, one in github.com private organization and the other in GHE. I have a situation and would like to get some feedback. 

We had some repositories in github.com private organization and those were configured with self-hosted renovate in github.com, now these repositories were migrated to GHE and we would like to point the renovate schema to GHE instead of github.com. 

My question is, do i need to remove renovate.json file from the repositories and onboard PR from GHE or just simply edit the renovate.json file in each repository and point the schema to GHE renovate

### Logs (if relevant)

<details><summary>Logs</summary>

```

Copy/paste the relevant log(s) here, between the starting and ending backticks

```

</details>

**Comment by rarkins** [comment]
The renovate.json is independent of which host it runs on. I don't understand what you mean by "point the schema"

**Reply by chandapukiran** [reply]
I am trying to understand the following scenario 

We have github.com private organization and self-hosted renovate running and onboarded all the repositories  
We have also GHE, it has self-hosted renovate running and onboard all the repositories created in GHE

Now, there are some repositories migrated from github.com to GHE and  i want to manage the migrated repositories using renovate hosted in GHE

Will there be new onboarding PR sent to the repositories? or because there is already a renovate.json file exist  , i can just update the schema URL 

We don't use the public schema https://docs.renovatebot.com/renovate-schema.json but kept it in the renovate-configure repo in both github.com and GHE

I am not sure if i am clear but happy to add more details

**Reply by rarkins** [reply, marked as answer]
It's fine to have a renovate.json file and you won't get another onboarding PR if so. Not recommended that you change the schema, but if you did then you need to take care of that
